sql/step_04_all_following_steps.sql

```diff
@@ -87,3 +87,4 @@ source step_90_login_retries.sql;
 source step_91_alter_laboratory_with_status.sql;
 source step_92_add_dicom_data_table.sql;
 source step_93_patient_consensus_table.sql;
+source step_94_remove_administrative_column.sql;
```

Comment:
There's no need of a new step since step_93 has never been released so far. We can modify step_93 and devs will align their DB.